uw-frame-components/portal/misc/partials/launch-button.html

```diff
@@ -0,0 +1,5 @@
+<a class="launch-app-button ng-scope"
+   target="{{ target }}"
+   ng-href="{{ href }}">
```

Comment:
I think that's a good idea given the current guidance, but I'm also wondering if we should revisit the guidance and tweak it a bit. [Links should make sense outside of their context](http://webaim.org/techniques/hypertext/link_text), so "Launch App" isn't really a helpful link when there are 10 other "Launch App" links on the page. "Launch Time & Absence," on the other hand, is clear in any context.

I'm all for adding an optional aria-label for the time being.